src/main/java/socialite/MainApp.java

```diff
@@ -75,22 +81,40 @@ public void init() throws Exception {
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         Optional<ReadOnlyAddressBook> addressBookOptional;
-        ReadOnlyAddressBook initialData;
+        ReadOnlyAddressBook initialAddressBook;
+        Optional<ReadOnlyCommandHistory> commandHistoryOptional;
+        ReadOnlyCommandHistory initialCommandHistory;
         try {
             addressBookOptional = storage.readAddressBook();
-            if (!addressBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample AddressBook");
+            if (addressBookOptional.isEmpty()) {
+                logger.info("Address book file not found. Will be starting with a sample AddressBook");
```

Comment:
Minor suggestion for Lines 90 and 94:

Do you want to simply conjoin 'Address' and 'book' rather than space them apart?  For example, you can consider stating "AddressBook file not found. Will be starting with a sample AddressBook". Might come across as more uniform/standardized in terms of our object naming.